contracts/src/v0.8/dev/KeeperRegistryLogic.sol

```diff
@@ -42,7 +46,13 @@ contract KeeperRegistryLogic is KeeperRegistryBase {
     PerformParams memory params = _generatePerformParams(from, id, performData, false);
     _prePerformUpkeep(upkeep, params.from, params.maxLinkPayment);
 
-    return (performData, params.maxLinkPayment, params.gasLimit, params.adjustedGasWei, params.linkEth);
+    return (
+      performData,
+      params.maxLinkPayment,
+      params.gasLimit,
+      params.fastGasWei * s_storage.gasCeilingMultiplier,
```

Comment:
nit. maybe add a comment here describing why you are multiplying. (i.e. this is nonExecution environment)

Reply:
Done